src/main/html/Part1Ex1.html

```diff
@@ -30,9 +30,12 @@
 Since a selector can match multiple elements, you can use tinymce.init to create multiple editors.
 TODO In the below script tag, instantiate TinyMCE editors in the below
 -->
-<textarea class="editorSet"></textarea>
-<textarea class="editorSet"></textarea>
-<script></script>
+<textarea id="editor2" class="editorSet"></textarea>
+<textarea id="editor3" class="editorSet"></textarea>
+<script>
+  tinymce.init({ selector: '#editor2' });
+  tinymce.init({ selector: '#editor3' });
```

Comment:
This could just be simplified to this for what it's worth 🤷 
```suggestion
  tinymce.init({ selector: '.editorSet' });
```